common/metadata-helper.h

```diff
@@ -0,0 +1,22 @@
+#pragma once
+
+#include <string>
+
+namespace rs2
+{
+    // Helper class that is responsible for enabling per-frame metadata on different platforms
+    // Currently implemented for Windows
+    class metadata_helper
+    {
+    public:
+        static metadata_helper& instance();
+
+        // Check if metadata is enabled using Physical Port ID
```

Comment:
I think it'd help readability if we gave an example of a port ID string here, and explained is can be retrieved with device::get_info(RS2_CAMERA_INFO_PHYSICAL_PORT)